src/routes/AppContent/index.tsx

```diff
@@ -75,6 +79,33 @@ const AppContent = () => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [activeAccountData]);
 
+    // Check if the account list currencies are in the client accounts currencies which is taken from OIDC tokens
+    // If not, request OIDC authentication
+    useEffect(() => {
+        if (accountList && isOAuth2Enabled) {
+            // Filter out disabled accounts to not trigger OIDC authentication
+            const filteredAccountList = accountList.filter(account => account.is_disabled === 0);
```

Comment:
nit: is `account.is_disabled` value guaranteed to be there?

Reply:
i checked the `account_list` api and `disabled` is always being returned